%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Seventh Chapter *********************************
%*******************************************************************************

\chapter{Concluding remarks}  %Title of the Seventh Chapter
\label{chapter7}

% from Paolo's thesis
% use Visscher et al 2017 (10 years of GWAS) instead
The success of the first genome-wide association studies, which uncovered common variants with moderate effect sizes for traits such as type 2 diabetes \cite{scott2007genome} and coronary artery disease \cite{wellcome2007genome}, fuelled the hope that GWAS could be applied to characterise the genetic component of virtually any trait of interest
\cite{visscher2012five}. 
% In parallel, 
In the years that followed, on the other hand, the GWAS approach has been the subject of numerous and evolving criticisms.
In particular,
researchers grew skeptical about GWAS, arguing
% as increasing numbers of traits have been surveyed using genetic association analyses, it has become increasingly clear that 
these expectations were too optimistic and that GWAS could only produce few associations,
variants identified could not explain estimated heritability \cite{manolio2009finding}, or that associations were spurious at all.
Today, there is now much more acceptance of the experimental design because the empirical results have been robust and overwhelming.
Indeed, GWAS actually informed a lot and bla bla bla \cite{visscher2012five}. 
Undeniably, the genotype-to-phenotype map has proven more complex than initially hypothesised. 
Many traits of interest are complex and can be regulated by tens or even hundreds of genetic loci with each locus having a small individual effect, polygenic models \cite{}
% (Wood et al., 2014; Ripke et al., 2014). 
and even omnigenic model \cite{boyle2017expanded}.
% Moreover, genetic variants commonly entail pleiotropic effects across several related phenotypes and diseases (Fortune et al., 2015; Pickrell et al., 2016) and interactions of genetic effects with environmental factors and other contexts are common (Andreassi, 2009; Winkler et al., 2015). 
\\

Additionally, many discovered variants lie in non-coding regions of the genome, hampering the interpretation of the molecular mechanisms that underlie such associations. 
Genetic studies of gene expression levels and other molecular traits have helped identify the regulated gene for a fraction of such intergenic GWAS loci. 
In particular eQTL mapping where common variants are associated with changes in gene expression have been very successful, with ...
However, as the effects of genetic variants can arise in specific tissue types or under specific stimuli \cite{gtex2015genotype, barreiro2012deciphering}, these genetic studies need to be conducted in disease-relevant cellular states. 
% \\
These are often hard to access, limiting studies to easily accessible tissues such as skin and blood, or post-mortem tissues, which have their own issues.
These tissues are also largely assessed using bulk RNA-seq, making it difficult to assess the relevant cell types, especially in complex tissues such as the brain.\\

Additionally, many (common) diseases have developmental component thus it is important to assess genetic regulation at those early stages of human development, which are impossible to access \textit{in vivo}.\\

Solution: \textit{in vitro} systems
Human iPSCs can be derived in a donor-specific manner, can be differentiated to virtually any cell type of interest.
Combined with single cell readouts,..\\

But

Challenge 1: iPSC differentiation protocols are not very efficient.

Challenge 2: methods to perform single cell eQTL mapping across cell types and states not there yet.\\

\section{Human iPSCs to model development and disease}

In work presented in this thesis, we attempted to quantify the efficiency of two distinct iPSC differentiation protocols.
In the first case, (chapter 4) short and sweet protocol, some differences between lines but not massive.
Causes: XCI already known, not enough power to determine genetic causes, no multiple lines per donor either, donor confounded with line.\\

In the second case (chapter 5) longer protocol, more lines.
Huge difference across lines in ability to make neurons, gene signature predictive of poor neuronal differentiation efficiency, possibly a sub-population of iPSCs.
Again, no power for genetics, no multiple lines per donor.
Some batch effects, but not enough to explain differences.
No correlation between the two protocols. \\

Room for improvement in our understanding on how and when the protocols work
Comparison between protocols, lineages, germ layers.
Are some lines just not going to differentiate no anything?
Or some lines are more prone to endoderm and consequently less so to ectoderm?
And is this dependent on the cell type of origin of iPSCs?
Is this a donor (genetic or other) effect or a line one?
i.e. trying other lines from a same donor could be a solution?
Or is this going to be limited to some people.
... are all questions that remain to be investigated in future studies.\\

Also challenge to verify similarity to corresponding \textit{in vivo} tissues.

\section{Bridging the genotype-phenotype gap}

Still a large gap in our understanding of function from genotype to phenotype.
Translation and clinical application still far away
One avenue to tackle these challenges is to increase statistical power by considering ever-increasing sample sizes. 
A complementary direction of investigation is to consider joint analyses across multiple traits, molecular layers and contexts.
eQTL mapping contributes to this by.. provides one layer of molecular understanding between genotype and phenotype \\

Single cell data has exploded recently, 
can be deployed to scale, combined with pooling strategies.
Can be used to estimate contexts

Adding cell-level context to eQTL mapping provides one more\\

1) identify pure cell population + cell population-specific eQTL

2) dynamic eQTL

3) other interaction eQTL

4) jointly (preliminary)\\

1 - contribution to best-practice (ch3)

also, discuss sharing of eQTL and cell-type specificity (ch4,5)

and disease-relevance (coloc, ch5)\\

2 - Contribution here by adding dynamic axis (ch4)
\cite{francesconi2014effects, strober2019dynamic}\\

3 - mention other interaction eQTL e.g. co-expression \cite{zhernakova2017identification}
using ASE \cite{knowles2017allele}\\

4 - The work described in Chapter 6 is preliminary, with further work required to confirm the reported findings, ..
Future directions related to this..



% \section{Key differences and commonalities between the two datasets}

% This section aims to highlight common aspects and differences between the two datasets described in chapters 4 and 5 respectively, and illustrate how they complement each other for our purposes.

% \subsection{iPSC lines used}
% In both cases, we used human iPSC lines from the HipSci consortium (see 1.4). 
% In fact, 45 lines were included in both studies.
% Through the HipSci resource, we also have access to bulk RNA sequencing for most of the same lines.\\

% % add table for numbers of lines in common?

% All HipSci lines are originated from fibroblasts, and we use all lines that were converted to a feeder-free medium (ref).
% Lines are from both sexes, and mostly healthy. 
% We do use a small portion of diseased lines in the endoderm differentiation protocol: 12 lines (out of 126) had monongenic diabetes, which did not affect any of the analyses. 

% \subsection{Experimental design}
% The pooling approach from the first study was replicated in the second, at a larger scale: in the endoderm protocol, 4 to 6 lines were differentiated together; in the midbrain neuronal differentiation protocol 9 to 21 lines were.
% In both cases, a subset of lines were differentiated in more than one pool (18/126, 35/215 respectively).\\

% An obvious difference is the length of the differentiation protocols.
% The definitive endoderm protocol is extremely short, lasting only 72h.
% As a consequence, considering different differentiation rates for different cells, we essentially have a continuos mapping of cells from pluripotent (0h i.e. day0) to definitive endoderm (72h i.e. day3).
% The midbrain dopaminergic neuronal differentiation protocol, on the other hand lasts nearly two months: 52 days.
% At the end, cells are much more differentiated and better resemble \textit{in vivo }mature cell types.
% We collected cells at day 11, day 30, and at the last time point, day 52.
% This time points are quite separate and discrete.


% \subsection{scRNA-seq technology used}

% Moreover, the scRNA-seq technology used in the two studies is different.\\

% In the endoderm differentiation study, for which data generation started in 2015, cells were sequenced using a plate-based scRNA-seq technology (SmartSeq2).\\

% The neuronal differentiation study is more recent (generation started in 2018) and thus used a technology that is more popular now (mainly because of UMI addition and scalability, see 1.3 for more detail), droplet-based (10X) sequencing.

% \subsection{Scale of study}
% A direct consequence of the technology used is a change in scale.
% SmartSeq2 is limited to 384 cell per plate
% Using 10X, one run can take up to 4,000 cells.
% We can observe this difference very clearly in our two datasets, made up by little less than 40,000 cells and just over a million (!) cells, respectively.\\

% We did also assess many more donors in the neuronal differentiation data: 215 (vs 125 from the endoderm study).

% \section{beyond the mean}

% In this chapter we want to show examples of eQTL mapping using different phenotypes, where we specifically exploit the single cell resolution.

% Interaction eQTL can be though of as a sort of GxE effect, where the effect of a genetic variant on the expression of a given gene is modulated by another factor.
% If in traditional (GWAS) GxE analysis the factor is often an environmental variable such as diet or exercise, in eQTL we are more often looking at molecular changes, such as cell type composition, or the level of activation due to some stimulus, etc..

% Using bulk:

% Sometimes, other genes can be used directly as factors, like in \cite{zhernakova2017identification}

% However, if we want to identify molecular changes it only makes sense to assess expression at the single cell level, and truly identify continuous states..

% \subsection{other results}

% As discussed in (2.4) there are multiple ways to test for interaction effects, where variation in the phenotype is affected by a combined effect of the genotype at a certain locus and some environmental component.\\

% In the context of eQTL mapping those environments are more often called cell states or types and are typically estimated from the expression profiles themselves. 
% Then an interaction eQTL might be an eQTL whose strength changes continuously over some differentiation trajectory for example, or an eQTL whose strength varies from cell type to cell type or from one cell cycle phase to another.\\ 

% In some sense, we have already performed a related analysis: by subdividing cells into developmental stages and cell types prior to eQTL mapping, we have essentially performed stratified tests (see section 2.4.1).
% In the neuronal differentiation study, we first assigned cells to cell types using marker genes, and then mapped eQTL in each cell type separately.
% In the endoderm study, we ordered our cells along a computationally inferred pseudotime, and then essentially binned cells in three groups, again reflecting known biology of this developmental process and using marker genes.\\

% However, the single cell resolution allows to assess more subtle changes, by performing a joint analysis on all cells and measuring the effect of differentiation taken as a continuum.
% In the next section, we show different approaches we used to perform interaction eQTL mapping in the endoderm differentiation data.

