%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Seventh Chapter *********************************
%*******************************************************************************

\chapter{Concluding remarks}  %Title of the Seventh Chapter
\label{chapter7}

The work described in Chapter 6 is preliminary

% from Paolo's thesis
% use Visscher et al 2017 (10 years of GWAS) instead
The success of the first genome-wide association studies, which uncovered common variants with moderate effect sizes for traits such as type 2 diabetes (Scott et al., 2007) and coronary artery disease (Burton et al., 2007), fuelled the hope that GWAS could be applied to characterise the genetic component of virtually any trait of interest \cite{visscher2012five}. 

However, as increasing numbers of traits have been surveyed using genetic association analyses, it has become increasingly clear that these expectations were too optimistic and that the genotype-to-phenotype map is more complex than initially hypothesised. 
Many traits of interest are complex and can be regulated by tens or even hundreds of genetic loci with each locus having a small individual effect (Wood et al., 2014; Ripke et al., 2014). 
Moreover, genetic variants commonly entail pleiotropic effects across several related phenotypes and diseases (Fortune et al., 2015; Pickrell et al., 2016) and interactions of genetic effects with environmental factors and other contexts are common (Andreassi, 2009; Winkler et al., 2015). 
Additionally, many discovered variants lie in non-coding regions of the genome, hampering the interpretation of the molecular mechanisms that underlie such associations. 
Genetic studies of gene expression levels and other molecular traits have helped identify the regulated gene for a fraction of such intergenic GWAS loci. 
However, as the effects of genetic variants can arise in specific tissue types or under specific stimuli (GTEx Consortium, 2015; Fairfax et al., 2014), these genetic studies need to be conducted in disease-relevant cellular states. \\


Access to cell types tricky

- \textit{in vitro} systems

\section{Human iPSCs to model development and disease}

Efficiency of iPSC differentiation protocols
Room for improvement in our understanding on how and when the protocols work
Comparison between protocols, lineages, germ layers.
Are some lines just not going to differentiate no anything?
Or some lines are more prone to endoderm and consequently less so to ectoderm?
And is this dependent on the cell type of origin of iPSCs?
Is this a donor (genetic or other) effect or a line one?
i.e. trying other lines from a same donor could be a solution?
Or is this going to be limited to some people.

\section{Bridging the genotype-phenotype gap}

Bridging the gap
Still a large gap in our understanding of function from genotype to phenotype.
Translation and clinical application still far away
Single cell data helps
from healthy to diseased

One avenue to tackle these challenges is to increase statistical power by considering ever-increasing sample sizes. 
A complementary direction of investigation is to consider joint analyses across multiple traits, molecular layers and contexts.

Contribution here by adding dynamic axis..

discuss sharing of eQTL and cell-type specificity


% \section{Key differences and commonalities between the two datasets}

% This section aims to highlight common aspects and differences between the two datasets described in chapters 4 and 5 respectively, and illustrate how they complement each other for our purposes.

% \subsection{iPSC lines used}
% In both cases, we used human iPSC lines from the HipSci consortium (see 1.4). 
% In fact, 45 lines were included in both studies.
% Through the HipSci resource, we also have access to bulk RNA sequencing for most of the same lines.\\

% % add table for numbers of lines in common?

% All HipSci lines are originated from fibroblasts, and we use all lines that were converted to a feeder-free medium (ref).
% Lines are from both sexes, and mostly healthy. 
% We do use a small portion of diseased lines in the endoderm differentiation protocol: 12 lines (out of 126) had monongenic diabetes, which did not affect any of the analyses. 

% \subsection{Experimental design}
% The pooling approach from the first study was replicated in the second, at a larger scale: in the endoderm protocol, 4 to 6 lines were differentiated together; in the midbrain neuronal differentiation protocol 9 to 21 lines were.
% In both cases, a subset of lines were differentiated in more than one pool (18/126, 35/215 respectively).\\

% An obvious difference is the length of the differentiation protocols.
% The definitive endoderm protocol is extremely short, lasting only 72h.
% As a consequence, considering different differentiation rates for different cells, we essentially have a continuos mapping of cells from pluripotent (0h i.e. day0) to definitive endoderm (72h i.e. day3).
% The midbrain dopaminergic neuronal differentiation protocol, on the other hand lasts nearly two months: 52 days.
% At the end, cells are much more differentiated and better resemble \textit{in vivo }mature cell types.
% We collected cells at day 11, day 30, and at the last time point, day 52.
% This time points are quite separate and discrete.


% \subsection{scRNA-seq technology used}

% Moreover, the scRNA-seq technology used in the two studies is different.\\

% In the endoderm differentiation study, for which data generation started in 2015, cells were sequenced using a plate-based scRNA-seq technology (SmartSeq2).\\

% The neuronal differentiation study is more recent (generation started in 2018) and thus used a technology that is more popular now (mainly because of UMI addition and scalability, see 1.3 for more detail), droplet-based (10X) sequencing.

% \subsection{Scale of study}
% A direct consequence of the technology used is a change in scale.
% SmartSeq2 is limited to 384 cell per plate
% Using 10X, one run can take up to 4,000 cells.
% We can observe this difference very clearly in our two datasets, made up by little less than 40,000 cells and just over a million (!) cells, respectively.\\

% We did also assess many more donors in the neuronal differentiation data: 215 (vs 125 from the endoderm study).

\section{beyond the mean}

In this chapter we want to show examples of eQTL mapping using different phenotypes, where we specifically exploit the single cell resolution.

Interaction eQTL can be though of as a sort of GxE effect, where the effect of a genetic variant on the expression of a given gene is modulated by another factor.
If in traditional (GWAS) GxE analysis the factor is often an environmental variable such as diet or exercise, in eQTL we are more often looking at molecular changes, such as cell type composition, or the level of activation due to some stimulus, etc..

Using bulk:

Sometimes, other genes can be used directly as factors, like in \cite{zhernakova2017identification}

However, if we want to identify molecular changes it only makes sense to assess expression at the single cell level, and truly identify continuous states..

% \subsection{other results}

% As discussed in (2.4) there are multiple ways to test for interaction effects, where variation in the phenotype is affected by a combined effect of the genotype at a certain locus and some environmental component.\\

% In the context of eQTL mapping those environments are more often called cell states or types and are typically estimated from the expression profiles themselves. 
% Then an interaction eQTL might be an eQTL whose strength changes continuously over some differentiation trajectory for example, or an eQTL whose strength varies from cell type to cell type or from one cell cycle phase to another.\\ 

% In some sense, we have already performed a related analysis: by subdividing cells into developmental stages and cell types prior to eQTL mapping, we have essentially performed stratified tests (see section 2.4.1).
% In the neuronal differentiation study, we first assigned cells to cell types using marker genes, and then mapped eQTL in each cell type separately.
% In the endoderm study, we ordered our cells along a computationally inferred pseudotime, and then essentially binned cells in three groups, again reflecting known biology of this developmental process and using marker genes.\\

% However, the single cell resolution allows to assess more subtle changes, by performing a joint analysis on all cells and measuring the effect of differentiation taken as a continuum.
% In the next section, we show different approaches we used to perform interaction eQTL mapping in the endoderm differentiation data.

