%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Third Chapter *********************************
%*******************************************************************************

\chapter{Mean level eQTL mapping using single cell RNA-seq data from iPSCs}
\label{chapter3}

% abstract

Traditional \gls{eqtl} mapping uses, as a measure of gene expression, bulk RNA-sequencing, where expression level is averaged across several cells from a given individual.
However, recent advances in experimental techniques allow to assess molecular phenotypes including gene expression at the level of single cells.
In particular, singe cell RNA-sequencing is now an established technique, with fairly standardised methods to perform both low-level analyses and complex tasks such as clustering and pseudotime estimation.
With the ability to identify cell types and states in an unbiased manner the use of \gls{scrnaseq} data is uniquely positioned to provide an extra layer to our understanding of the genetic regulation of expression across a plethora of cell types and states.
As a consequence, single cell \gls{eqtl} mapping (where scRNA-seq profiles are combined with genotype information to map \gls{eqtl}) is quickly emerging as a field and it promises to greatly improve our understanding of the architecture of human disease across tissues.
As more and more studies emerge, it becomes important to establish a `best practice' pipeline to optimise yield of sc-\gls{eqtl} studies and uniform methods across the field.
Here, we use matched bulk and single cell RNA-seq from around 100 human \gls{ipsc} lines to assess our ability to detect \gls{eqtl} using single cell RNA-seq data and to replicate \gls{eqtl} identified using bulk RNA-seq.
We systematically assess the role played by multiple factors including the chosen aggregation method and covariates used and compare results across technologies.
We believe this is a useful resource for several future studies that will want to perform eQTL mapping using single cell expression profiles.

\newpage

% \begin{Comment2}
% % \subsection{Contributions}
% \hspace{-3mm}\textbf{Contributions} 
% % This work was done in collaboration between the Stegle, Bonder and McCarthy labs.
% The project was lead by Giordano Alvari under Marc Jan Bonder's supervision and mine.
% % Christina Azodi supervised by Davis McCarthy performed the simulation analyses.
% % \vfill
% \end{Comment2}

% \newpage

\section{Introduction}


% % Intro on molecular readouts as intermediate to understand genotype-phenotype mechanisms.
% % Going back in time to our progressive better understanding of molecular machinery, starting from Crick postulating the central dogma.

% % \subsection{Estimation of gene expression levels}

% % In this thesis, we focus on gene expression, i.e., the transcriptome, as a molecular phenotype.
% % In general, the transcriptome describes the complete set of transcripts in a tissue or cellular sample, and their respective quantity. 
% % As a precursor of protein expression, mRNA can serve as a proxy of gene expression levels. 
% % Multiple approaches have been developed to measure cellular mRNA levels, including hybridization- and sequencing-based approaches. 
% % In the case of hybridization-based methodology, reverse transcription (RT) is used to generate a complementary \gls{dna} (c\gls{dna}) template of the mRNA. 
% % When this c\gls{dna} template is being amplified with labelled hybridization probes via quantitative polymerase chain reaction (qPCR), fluorescence is emitted according to the oligonucleotides that are being incorporated. 
% % Based on the fluorescence signal, the genetic sequence of the original mRNA strand can be reconstructed. 
% % Alternatively, a hybridization microarray contains pre-defined probes for transcripts of every known gene of one or several species. Transcripts that are not known a priori can be detected with tag-based methods such as SAGE (Serial Analysis of Gene Expression); SAGE uses small tags that cover only fragments of a transcript as probes, and can therefore, opposed to hybridization microarray chips, also discover transcripts whose full sequence is unknown. 
% % However, a large proportion of the tags used by SAGE does not map to unique regions of a reference genome due to their short length, and can therefore not be used for transcript quantification. 
% % Further, tag-based approaches do not ensure the analysis of the entire transcriptome, and can generally not discover alternative splicing events (Wang et al., 2009).
% % RNA sequencing (RNA-Seq) based on next-generation sequencing (NGS) of the c\gls{dna} allows for genome-wide quantification of the transcriptome. After obtaining one (single-read RNA-Seq) or two paired (paired-end RNA-Seq) sequence reads per c\gls{dna} fragment, the sequencing reads are either aligned to a reference genome or are assembled de novo. 
% % From the number of RNASeq reads that map to a particular gene an estimation of gene expression can be deduced. 
% % In this thesis, we use FPKM (fragments per kilo base per million reads mapped) for gene expression estimations. 
% % FPKM quantify the number of reads that are assigned to a given gene, normalised by gene length and the total sequencing depth (Wang et al., 2009).
% % Opposed to the hybridization- and tag-based methods, NGS allows for identifying completely new genes, previously unknown genetic variants in the genes, variation in alternative splicing, or post-transcriptional modifications. In addition, RNA-Seq can quantify the vast array of non-coding RNA molecules (Section 1.1.1). Altogether, sequencing-based assessments of the transcriptome deliver more detailed insights into gene expression variability than hybridization- or tag-based approaches.
% % In this thesis, we quantify gene expression by RNA-Seq measurements of mRNA levels.

% % \subsubsection{From microarrays to (bulk) RNA-sequencing}

% % Useful for comparative transcriptomics, e.g. samples of the same tissue from different species.
% % Useful for quantifying expression signatures from ensembles, e.g. in disease studies.
% % Insufficient for studying heterogeneous systems, e.g. early development studies, complex tissues (brain)stuart2019integrative
% % Does not provide insights into the stochastic nature of gene expression

Super briefly intro on gene expression estimation, first micro-arrays then bulk RNA-seq.

% \subsection{scRNAseq}
\subsection{The `resolution revolution'}
\label{sec:scrnaseq}

The first single-cell RNA sequencing (scRNA-seq) experiment was published in 2009, and the authors profiled only eight cells \cite{tang2009mrna}. 
Only 7 years later, 10X Genomics released a dataset of more than 1.3 million cells \cite{102016our}.
Today, over 1,000 scRNA-seq datasets have been published 
\cite{svensson2018exponential, svensson2019curated, svensson2020single},
using a number of different technologies (Fig. \ref{fig:scrnaseq_technologies}).
By now scRNA-seq is an established technique and methods are available to efficiently and reliably perform low-level analyses such as read alignment, cell calling and \gls{qc} as well as workflows to perform higher level tasks such as batch correction, normalisation, clustering and pseudotime inference.\\

In some cases, those methods have been directly borrowed from bulk RNA-sequencing methods; other times, methods tailored specifically for single cell data were proposed \cite{stegle2015computational}.
A typical workflow for single cell RNA-seq data implemented in R can be found on Bioconductor using several scRNA-seq specific R packages \cite{lun2016step, risso2016scrnaseq, mccarthy2017scater, lun2019singlecellexperiment}\footnote{at https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html and

https://osca.bioconductor.org}.

% Here I will only mention a few key steps.

\begin{figure}[h]
\centering
\includegraphics[width=15cm]{Chapter1/Fig/scrnaseq_technologies_svensson2018.jpg}
\caption[scRNA-seq technologies]{\textbf{Scale of scRNA-seq experiments}.\\
Technologies that have allowed....

% make own version adding SmartSeq3 etc 

adapted from \cite{svensson2018exponential}}
\label{fig:scrnaseq_technologies}
\end{figure}

% \subsubsection{Low-level analysis}
% reads QC 
% alignment
% mapping QC

% cell QC (e.g. remove cells with less than xx total counts, yy total genes)
% possibly deal with doublets etc - in our case, donor assignment is also here
% normalization (account for differences due to read coverage etc)
% log transformation (variance stabilising)

% feature selection (isolate most informative genes, e.g. highly variable genes - HVGs)
% genes that behave differently from your expected mean-variance relationship



% Several platforms implement the entire processing workflow, or at least large portions of it.
% These include R packages seurat \cite{stuart2019comprehensive} and SINCERA (SINgle CEll RNA-seq profiling Analysis, \cite{guo2015sincera}) and python package scanpy \cite{wolf2018scanpy}. 



% \subsection{Computational modelling of scRNA-seq}

% Analysis of scRNA-seq data requires a new set of considerations, largely concerning technical signals, that were not relevant for bulk RNA-sequencing work. 
% Moreover, the resolution of this single-cell data also allows a number of more powerful analysis techniques to be applied.
% This section describes, in brief, how a typical single-cell RNA-sequencing dataset may be analysed.



% \subsubsection{Normalization and batch correction}

% Count matrix
% 10 Genomics: UMI counts
% Smartseq2: expected counts or TPM (similar to bulk)



% \textbf{feature selection} (isolate most informative genes, e.g. highly variable genes - HVGs)\\

% genes that behave differently from your expected mean-variance relationship

% optional: centering+scaling - standardizing

% batch correction (stronger than normalisation) 
% mutual nearest neighbours (MNN, \cite{haghverdi2018batch}) - and then fastMNN
% canonical correlation analysis (CCA, implemented in Seurat \cite{butler2018integrating}), Stuart et al 2019
% LIGER iNMF (negative matrix factorization),
% Harmony (\cite{nowotschin2019emergent}) - iterative soft k means (fastest)
% Welch et al 2019, Korsunsky et al 2019


% \subsubsection{Computational analysis}

% \textbf{dimensionality reduction}

% \gls{pca} was first introduced by Pearson over a hundreds years ago (\cite{}, see section 1), yet remains one of the most widely used tools \\

% \textbf{clustering}

% unsupervised\\

% \textbf{pseudotime}

% PCA, diffusion maps\\

% \textbf{DE}

% DESeq, edgeR



% \subsubsection{Visualization techniques}

% Even after application of a dimension-reduction procedure, a typical dataset will retain more than three biologically important dimensions in its new subspace, which makes visual representation of the data challenging. 

% Transforming high-dimensional data into a human-readable format is therefore an important challenge for single-cell data interpretation.

% scRNA-seq data visualization techniques used in this thesis: 



% \begin{itemize}
%     \item \gls{pca}
%     \item t-distributed stochastic neighbour embedding (tSNE) \cite{maaten2008visualizing}
%     \item uniform manifold approximation and projection (UMAP) \cite{mcinnes2018umap}
% \end{itemize}



\subsection{Single cell eQTL mapping}

With the ability to identify cell types and states in an unbiased manner the use of scRNA-seq data is uniquely positioned to provide an extra layer to our understanding of the genetic regulation of expression across a plethora of cell types and states.
As a consequence, single cell eQTL mapping has emerged as a field and promises to improve our understanding of genetic regulation both in health and disease across tissues \cite{wills2013single, van2018single, kang2018multiplexed, sarkar2019discovery, cuomo2020single, jerber2020population, van2020single1}.
As more and more studies emerge, it becomes important to establish a `best practice' pipeline to optimise yield of sc eQTL studies and uniform methods across the field.\\

Here, we leverage bulk and single cell gene expression of matched human \gls{ipsc}s lines from around 100 donors to identify general guidelines for \gls{eqtl} mapping using scRNA-seq data.
We compare several manners of normalising and aggregating expression across cells per donor as well as different models to test for \gls{eqtl} and compare to equivalent results obtained when using bulk RNA seq data.
Whilst for most individuals we have plate-based sc-RNAseq data (SmartSeq2, \cite{picelli2013smart}), we also have data using the 10X Genomics platform \cite{zheng2017massively} for a subset of around 30 samples, which allows us - to an extent - to also compare results across single cell technologies.\\

\section{What is different in single cell data?}

When we perform \gls{eqtl} mapping, we are interested in finding differences in expression level between individuals, when stratified by their genotypes at specific genomic loci. 
Under the assumption that we are looking at an otherwise homogeneous population of cells (e.g. all cells are from the same cell type) it is reasonable to consider the sum or the average expression for each individual, across all cells.
When we use bulk RNA sequencing expression profiles, that is essentially what happens: all cells from an individual are pooled, the mRNA is extracted and sequenced. 
The resulting reads are then mapped onto a reference genome, and the expression level of each gene is quantified as the number of reads obtained from one donor that uniquely map to that gene. 
A bulk RNA-seq experiment, therefore, results in one individual measure of `abundance' of each gene for each donor. 
Such measure is the results of aggregating over XX-YY cells (ZZ on average, \cite{}) and, at least for expressed genes (average TPM > YY) the vector of gene expression across individuals follows a distribution that can be approximated as Gaussian.\\

% The intuition here is that 
% Pool of RNA transcripts from many genes (low probability for a given gene), get a sample to sequence. 
% Poisson: sampling from large n, small p (samples are technical replicates)
% (biological replicates - NB > Poisson, larger variance )

% As discussed in section 1.3 of the Introduction, 
Recent advances in experimental technologies have provided robust methods for single-cell RNA sequencing (scRNA-seq), allowing to assay the genome-wide transcriptome of hundreds to thousands of individual cells. 
Whilst scRNA-seq data provides increased resolution and promises great insights into cellular function, the data also are much sparser, and the amount of cells that can be assayed for an individual is limited compared to bulk (on average XX compared to YY for bulk RNA-seq, \cite{}).
Moreover, in addition to observing a smaller number of cells (and therefore total reads) for an individual as compared to bulk, the read distribution typically does not follow a normal distribution, mostly due to very different amounts of reads coming from different donors. \\

add table/figure here
% This is in part due to the `double sampling' that is inherent of the technology: there is a chance of not sampling any reads from one gene in one cell, and there is a chance of not sequencing any reads from that cell at all.

% Add differences in number of total reads, read distribution, describe “double sampling” process.

\section{Methods}

In order to produce bulk-like results, two main approaches can be used.

Under the assumption that we are looking at a single cell type, we can i) aggregate counts across all cells from an individual (e.g. taking the average expression value) and generate a `pseudo-bulk' measure, and then run the test exactly as if it were bulk; or ii) use full single cell expression, without aggregating.

% In this chapter, we introduce the two datasets that we will use throughout the thesis, setting the scene for all other chapters. 
% These are among the very first datasets of their kind, assessing single cell gene expression profiles for hundreds of genetically diverse individuals, allowing to interrogate the effect of common genetic variation on transcriptional variability.
% Previously, most \gls{eqtl} mapping studies in humans were performed using bulk RNA-sequencing, and most scRNA-seq studies were performed in a handful of individuals only, or in model organisms (often also limited to a few strains).
% Notably, one study performed \gls{eqtl} mapping using scRNA-seq \cite{van2018single} recently in blood cells. 
% However, this data is limited to 45 individuals and to a single time point, lacking the differentiation axis of our studies.

% \subsection{Datasets}

% \begin{itemize}
%     \item iPS data
%     \item simulated data
% \end{itemize}

\subsection{Data}

We use iPS data from \cite{cuomo2020single} - day0 only, etc

Smartseq2 \cite{picelli2013smart} data from 10,000 (get exact number) cells from 112 unique unrelated donors, across YY differentiation experiments. 

\subsection{Aggregation strategies}

Pseudobulk approach.
To comprehensively investigate which aggregation method would result in most power to to identify single cell eQTL, we tried:

\begin{itemize}
    \item Mean: average expression across cells from each donor and sequencing run
    \item Median: median expression across cells from each donor and sequencing run
    \item Sum: summed expression across cells from each donor and sequencing run
    \item Total mean: average expression across all cells from each donor, aggregated across sequencing runs
    \item Total sum: similar to the total mean, but summing reads instead
\end{itemize}

Aggregation is done not only at the donor level but also for each individual sequencing run (i.e. all iPS cells from a given donor in a single sequencing run) in the mean, median and sum.
This is done in an attempt to account for batch effects (needs explaining).
For what we call `total' sum and mean, on the other hand, aggregation (i.e. sum or mean) is done per donor only, to maximise numbers of cells per donor.\\

Normalisation of the scRNA-seq data was performed in two different ways depending on the aggregation method used.
For mean and median aggregation of expression data, we first performed standard single cell data normalisation using scran/scater normalisation.
Briefly, log2(cpm+1) using size factors..
The mean and the median were calculated on the resulting normalised counts (Fig. \ref{fig:sc_qtl_workflow}).

On the other hand, summed count values were obtained directly from the un-normalised data.
Normalisation was then applied on the resulting pseudo-bulk counts, using methods typically used for bulk RNA-seq data.
In particular, edgeR... (Fig. \ref{fig:sc_qtl_workflow}).\\

% add workflow figure

\begin{figure}[h]
\centering
\includegraphics[width=15cm]{Chapter3/Fig/sc_qtl_workflow.png}
\caption[sc-eQTL workflow]{\textbf{sc-eQTL workflow}.\\
Placeholder: different approaches tested to perform \gls{eqtl} mapping using scRNA-seq profiles}
\label{fig:sc_qtl_workflow}
\end{figure}

\newpage

\subsection{Covariates}

Expression covariates PCs, PEER established (add references, e.g. GTEx..)

Another parameter we varied was the type and number of expression covariates included in the model to account for global expression variation (see section 2.X).

In particular, we computed principal components (PCs) from the (aggregated) expression matrix.
We also computed 10 MOFA factors \cite{argelaguet2018multi} as well as XX PEER factors \cite{stegle2010bayesian,stegle2012using}.

Since the (total?) mean performed best as an aggregation method, we tested various number of covariates for this model only.

In particular, we included 5, 10, 20 and 50 PCs in the model as covariates and evaluated performance.

% \section{Results in iPSCs}
\section{Results}

\subsection{Comparison partners}

\begin{itemize}
    \item bulk RNA-seq with matched samples (i.e. individuals for which we have both bulk and sc-RNAseq data, N = 100)
    \item bulk RNA-seq all samples (i.e. all samples for which we have bulk RNA-seq data, N = 300)
    \item matched 10x scRNA-seq (for a subset of common samples, N = 30)
\end{itemize}


First, we tested for associations between common genetic variants and gene expression at \gls{ipsc} stage. 
Briefly, for each donor, experimental batch, and differentiation stage, we quantified each gene’s average expression level, before using a linear mixed model to test for \textit{cis} \gls{eqtl}, adapting approaches described above and used for bulk RNA-seq profiles (+/- 250kb, MAF > 5\% \cite{kilpinen2017common}). 

\begin{equation}
    \boldsymbol{\mu} = \sum_i^{10}\alpha_i \mathbf{PC}_i + \mathbf{g}\beta + \mathbf{u} + \boldsymbol{\psi}  
\end{equation}

This identified 1,833 genes with at least one \gls{eqtl} (denoted eGenes; FDR <10\%; 10,840 genes tested; Supplementary Data 3). 

% \section{Replication in bulk, 10x}

To validate our approach, we also performed \gls{eqtl} mapping using deep bulk RNA-sequencing profiles from the same set of \gls{ipsc} lines (“iPSC bulk”; 10,736 genes tested) generated as part of the \gls{hipsci} project \cite{kilpinen2017common}, yielding consistent \gls{eqtl} (~70\% replication of lead \gls{eqtl} effects; nominal p value < 0.05).\\ 

These \gls{ipsc} \gls{eqtl} were further confirmed by analysis of scRNA-seq data generated from a subset of 5 experiments using a droplet-based approach.


% \section{Results in simulated data}

\section{Discussion}

% from endodiff paper
These results illustrate a difference between bulk and single-cell transcriptomics, as applied to eQTL mapping: the trade-off between statistical power and cellular resolution. 
In our analysis of iPSC, bulk transcriptomes provided higher statistical power for
discovery of eQTL. 
However, as we have demonstrated, a single-cell approach allows detailed annotation of changing eQTL effects across heterogeneous cell types and cell states, with the ability to better interpret the context-specific role of individual genetic
variants. 
As single-cell approaches are extended to more disease-relevant tissues and cell types, this may provide important clues on the causal role of genetic variants in disease. 
The single-cell technology employed also has implications for what can be assessed. 
While we found that similar eQTL signals could be detected with both Smart-seq2 and 10x approaches, the full-length transcripts of Smart-seq2 allowed quantification of ASE, which is not possible with the 3' fragments sequenced using the 10x protocol.
A further advantage of the application of single-cell transcriptomics in this study was to enable the pooling strategy. 
While the feasibility of pooling samples has previously been demonstrated for peripheral blood mononuclear cells\cite{kang2018multiplexed}, we have extended this to cell lines differentiated together in culture. 
This provided higher throughput, and enabled the characterisation of intrinsic line-to-line variation in differentiation efficiency in a controlled setting. 
While the endoderm differentiation protocol considered here is short and efficient, other protocols (e.g., to generate neurons\cite{tao2016neural}) are much more challenging, making a pooling strategy useful for scaling up these protocols to population-scale.\\


Is it relevant to do? 
vs gating+bulk


Comparison with bulk

trade off: power vs resolution
also pooling allowed using single cells

Comparison across technologies

plate-based vs droplet-based

