%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Third Chapter *********************************
%*******************************************************************************

\chapter{Population-scale single cell maps of differentiating human iPS cells}

In this chapter, we introduce the two datasets that we will use throughout the thesis, setting the scene for all other chapters. These are among the very first generated datasets of their kind, combining single cell RNA-seq profiling at population scale, allowing to interrogate the effect of common genetic variation on transcriptional variability.

\section{iPSC differentiation toward an endoderm fate}

The first dataset we describe is genome-wide single cell RNA-sequencing profiling of nearly 40,000 differentiating human iPS cells collected from cell lines from over one hundred healthy donors. Data is collected at four time points, at pluripotent state (day0), and then every 24h along differentiation to a definitive endoderm cell fate (day3).

\vspace{5mm}

\subsection{Contributions}

\noindent The dataset description and the analyses included in this chapter are largely contained in (Cuomo et al. 2020).

\vspace{5mm}

\noindent The data was generated by Ludovic Vallier’s lab at the Sanger Institute, and the experiments were largely led by Dr. Mariya Chhatriwala, who also contributed to the interpretation of the results. Cell lines from the HipSci project were used.
The statistical methods and analyses described in this chapter were co-supervised by Oliver Stegle and John Marioni. I processed the data and performed lengthly quality control (QC) in collaboration with Dr. Davis McCarthy. I developed and implemented the statistical methods under the supervision of Oliver Stegle and John Marioni with some input from Dr. Daniel  Seaton. The code for processing, analysing and plotting the data is open source and freely accessible here: https://github.com/annacuomo/singlecell\_endodiff\_paper. 

\vspace{5mm}

\noindent The paper was written in collaboration with Daniel Seaton, John Marioni and Oliver Stegle, and is available at https://www.nature.com/articles/s41467-020-14457-z.

\subsection{Introduction}

As highlighted in section XX, the early stages of embryogenesis involve dramatic and dynamic changes in cellular states. As cells go from a pluripotent state where they still have the potential to differentiate to any (embryonic) cell type, to committing to a specific cell fate, many molecular programs and mechanisms are activated and tightly regulated.

Our understanding of such mechanisms is still only partial, yet a lot has been learnt in model organisms including fruit flies, Zebrafish, and mice [REF]. For obvious (ethical) reasons, such mechanisms cannot be studied \textit{in vivo} for humans. There exist a few studies that use [aborted foetuses] but the data is hard to access, often limited to a narrow time frame within development, etc...

Human induced pluripotent stem cells (iPSCs) and iPS-derived cells offer great potential to interrogate cell types and states that are challenging if not impossible to access in human, \textit{in vivo}.

In particular, the study of early development can be mimicked \textit{in vitro} using iPS-derived differentiation protocols. This highly controlled setup allows us to sample at tight intervals in time and provides a unique opportunity to study the dynamic effect of common genetic variants on gene expression regulation during early development.

However, iPSC differentiation protocols are challenging to apply in practice: most protocols generate much more diversity than intended in terms of cell types (ref). Additionally, extensive batch to batch as well as line-to-line heterogeneity has been observed (refs). Finally, the protocols are lengthy and hard to scale leading to limited throughput. 
 
In this study, we employed different strategies to combat some of these issues. 

First, the single cell RNA-seq readout allows cellular heterogeneity to be assessed in a continuous manner across development.
Second, our pooling strategy - where we differentiate cells coming from 4 to 6 donors in the same pool - allows us to increase throughput to population-scale and allows us to control for technical batch to batch variation, enhancing the ability of inter line comparisons.
Finally, we selected a short and efficient protocol, which models an extremely well understood developmental process thus acting as a proof of principle study.

In this study, we differentiate 125 iPS cell lines towards the definitive endoderm, one of the three germ layers together with ectoderm and mesoderm. This is a short (three days) and well-established protocol that is the basis for longer differentiations toward disease-relevant tissues such as the gut, the pancreas and the lungs.

The generation of population-scale collections of differentiating human iPSCs allows the study of inter-individual variability effects, which is key as cellular reprogramming becomes an increasingly used tool in molecular medicine.


\subsection{Data processing and QC}


\subsubsection{Demultiplexing donors from pooled experiments} 

Our pooled experimental design means that cells from multiple donors are differentiated together in the same experiment. To be able to link the genetic background of an individual with their transcriptional profile we need to map the cells back to their donor of origin, without the use of any barcode.

Indeed, we find that for the large majority of cells the RNA-seq reads map to a sufficient number of common genetic variants for us to reliably assign each cell to its original donor.

In particular, assignment of cells to donors was performed using Cardelino (ref). Briefly, Cardelino estimates the posterior probability of a cell originating from a given donor based on common variants in scRNA-seq reads, while employing a beta binomial-based Bayesian approach to account for technical factors (e.g. differences in read depth, allelic drop-out, and sequencing accuracy). For this assignment step, we considered a larger set of n = 490 HipSci lines with genotype information, which included the 126 lines used in this study. A cell was assigned to a donor if the model identified the match with posterior probability >0.9, requiring a minimum of 10 informative variants for assignment. Cells for which the donor identification was not successful were not considered further. Across the full dataset 99pct of cells that passed RNA QC steps (below) were successfully assigned to a donor.

\subsubsection{scRNA-seq feature quantification and quality control}

Single cell profiles were obtained using the SmartSeq2 technology (ref). This is a plate-based technology that involves single cells being sorted in 384 independent wells on a plate (section XX of the Introduction). 

Obtained reads are then mapped to a human genome reference. Gene-level expression quantification was performed using Salmon (ref). Briefly, Salmon quantifies transcript level expression levels, similar to Kallisto (ref). Then, such values are summarised at a gene level (estimated counts per million (CPM).

We performed quality control (QC) of scRNA-seq profiles following a widely used pipeline using Bioconductor (ref) packages scater and scran, implemented in R (refs).  

In particular, cells were retained for downstream analyses if they had at least 50,000 counts from endogenous genes, at least 5,000 genes with non-zero expression, less than 90pct of counts came from the 100 highest-expressed genes, less than 15pct of reads mapping to mitochondrial (MT) genes, they had a Salmon mapping rate of at least 60pct, and if the cell was successfully assigned to a donor. 

\subsubsection{Flow cytometry}

FACS: fluorescence-activated cell sorter.

Live/Dead (7AAD)
Dead cells as identified based on 7AAD staining were discarded. 

Single cells/Doublets

CXCR4/Tra160 
Protein surface markers Tra-1-60 (iPS) and CXCR4 (definitive endoderm)

\subsubsection{scRNA-seq processing}

SmartSeq2 data does not include unique molecule identifiers (UMIs) which can help normalisation by providing a comparable measure of the amount of reads sequenced per cell. In the absence of UMIs, we can borrow information from cells with similar total number of reads and correct for overall library size. Such size factor normalisation of counts was performed using scran (ref). 

Expressed genes with an HGNC symbol were retained for analysis, where expressed genes in each batch of samples were defined based on (i) raw count >100 in at least one cell prior to QC and (ii) average log2(CPM+1) >1 after QC. Normalised CPM data were log transformed (log2(CPM+1)) for all downstream analyses. The joint dataset was investigated for outlying cell lines or experimental batches, which identified no clear groups of outlying cells (Supplementary Fig. 21, 22). 

As a final QC assessment, we considered possible differences between cell lines from healthy and diseased donors. In particular, a subset of 11 cell lines were derived from neonatal diabetes patients, and differentiated together with cell lines from healthy donors across 7 experiments (out of 28). There was no detectable difference in differentiation capacity between healthy and neonatal diabetes lines in these experiments (P > 0.05), and cells from both sets of donors overlapped in principal component space (Supplementary Fig. 23). Thus, we included cells from all donors in our analyses irrespective of disease state.

The final merged and QC’ed dataset consisted of 36,044 cells with expression profiles for 11,231 genes (Supplementary Figs. 2 and 5).


\subsection{Results}

\subsubsection{Data overview}

 We considered a panel of well-characterized human iPSC lines derived from 125 unrelated donors from the Human Induced Pluripotent Stem Cell initiative (HipSci) collection (Kilpinen et al. 2017). In order to increase throughput and mitigate the effects of batch variation, we exploited a novel pooled differentiation assay, combining sets of four to six lines in one well prior to differentiation (28 differentiation experiments performed in total; hereon “experiments”; Fig. 1A). Cells were collected at four differentiation time points (iPSC; one, two and three days post initiation - hereon day0, day1, day2 and day3) and their transcriptomes were assayed using full-length RNA-sequencing (Smart-Seq2 (Picelli et al. 2014)) alongside the expression of selected cell surface markers using FACS (TRA-1-60, CXCR4). Following quality control (QC), 36,044 cells were retained for downstream analysis, across which 11,231 genes were expressed.
 (Fig.1a)
 
 Exploiting the observation that each cell line’s genotype acts as a unique barcode, we demultiplexed the pooled cell populations, enabling identification of the cell line of origin for each cell (similar to(Kang et al. 2018)). At each time point, cells from between 104 and 112 donors were captured, with each donor being represented by an average of 286 cells (after QC). The success of the differentiation protocol was validated using canonical cell-surface marker expression: consistent with previous studies (Chu et al. 2016), an average of 72pct of cells were TRA-1-60(+) in the undifferentiated state (day0) and an average of 49pct of cells were CXCR4(+) three days post differentiation (day3).

\subsubsection{Sources of variation} 

Variance component analysis across all genes (using a linear mixed model) revealed the time point of collection as the main source of variation, followed by the cell line of origin and the experimental batch (Fig. 1B). Consistent with this, the first Principal Component (PC, calculated from the top 500 highly variable genes) was strongly associated with differentiation time (Fig. 1C), motivating its use to order cells by their differentiation status (hereafter “pseudotime”, Fig. 1C). Alternative pseudotime inference methods yielded similar orderings.

\subsubsection{Developmental stages}
 
Critically, the expected temporal expression dynamics of marker genes that characterise endoderm differentiation was captured by the ordering of cells along the inferred pseudotime (Fig. 1D). Exploiting these markers of differentiation progress and pseudotime, we assigned 28,971 cells (~80pct) to one of three canonical stages of endoderm differentiation: iPSC, mesendoderm (mesendo) and definitive endoderm (defendo) (Fig. 1C). A smaller fraction of cells (N = 7,073) could not be confidently assigned to a canonical stage of differentiation; these cells were heavily enriched for those collected at day2, when rapid changes in molecular profiles are expected, reflecting a transitional population of cells.

\subsubsection{Map to gastrulation atlas??}


\section{iPSC differentiation toward dopaminergic neurons}

The second dataset we describe is genome-wide single cell RNA-sequencing profiling of over 1Million differentiating iPS cells collected from cell lines from over two hundred healthy donors. Data is collected at three maturation stage following differentiation to midbrain dopaminergic neurons: progenitor-like state (day11), young neurons (day30), and more mature neurons (day52). Additionally, just before the latest time point half of the cells were stimulated with rotenone, to simulate oxidative stress. 

\vspace{5mm}

\subsection{Contributions}

The dataset description and the analyses included in this chapter are largely contained in (Jerber*, Seaton*, Cuomo* et al. 2020).

\vspace{5mm}

The data was generated by Dan Gaffney’s lab at the Wellcome Trust Sanger Institute, and the experiments were largely led by Dr. Julie Jerber, who also contributed to the interpretation of the results. The statistical methods and analyses described in this chapter were co-supervised by Dan Gaffney and Oliver Stegle. Dr Daniel Seaton processed the data and performed lengthly quality control (QC), with my help. Daniel and I also developed and implemented the statistical methods under the supervision of Oliver Stegle and Dan Gaffney with some input from John Marioni and Natsuhiko K. The code for processing, analysing and plotting the data is open source and freely accessible here: https://github.com/xxx.

\vspace{5mm}

The paper was written in collaboration with Julie Jerber, Daniel Seaton, Dan Gaffney and Oliver Stegle, with input from Natsuhiko K. and John Marioni whilst a preprint can be found on biorxiv: XX.


\subsection{Introduction}

longer iPSC differentiation protocol

relevant for cell therapy (dopaminergic neurons and PD)

different 

\subsection{Data processing and QC}

\subsection{Results}


\section{Key differences and commonalities between the two datasets}

iPSC lines (from HipSci), XX in common.
feeder free, ecc.

Pooling approach similar (though different scale)

continuous vs discrete (and short vs long, i.e. day3 vs day52)

plate-based (SmartSeq2) vs droplet-based (10x) sequencing
-> scale! (40k cells vs a million!)
different number of donors, too (125 vs 215).



