%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Sixth Chapter *********************************
%*******************************************************************************

\chapter{LMM framework for multi-environment interaction eQTL using scRNA-seq}

%********************************** %First Section  **************************************

\section{Introduction} 

Population-scale cohorts that combine genotyping with single cell expression profiles have fostered interest to study the effect of common genetic variants on gene regulation at single cell resolution.\\

It has already been demonstrated that single cell expression quantitative trait loci (eQTL) mapping can be achieved, and that known eQTL previously identified using bulk expression can be replicated both in iPS cells and blood \cite{cuomo2020single,van2018single}.
Additionally, the single cell resolution allows for the interrogation of interaction eQTL, where the strength of the genetic effect of eQTL is modulated by the cells’ types and states \cite{cuomo2020single}. 
These kinds of studies can add a layer of interpretability of the role of genetic variants on gene expression regulation. 
Analyses of eQTL context-specificity have been so far limited to a single state variable (e.g. cell type, \cite{fairfax2012genetics} or stimulation status,\cite{fairfax2014innate} ) and individual genetic variants.\\

A recently proposed method called StructLMM (structured linear mixed model, described in 2.4.3) allows for the robust joint analysis of GxE effect of multiple environmental variables \cite{moore2019linear}.
This model can be applied, with some care, to the mapping of interaction eQTL, where genes’ single cell expression profiles are the tested outcomes, and the cells’ states and types are the environmental variables. 
The main limitation of StructLMM is that it does not allow to fully control for the intrinsic population structure of the tested samples [REF]. 
When using single cell measurements, we are in practice dealing with genetically identical replicate observations for a donor. 
The relatedness structure between samples (or cells in this context) is therefore non-trivial, as the cell measurements are not independent, as assumed by the model.\\

Here, we propose sc-StructLMM, an extension of the StructLMM model that builds on it while allowing control for confounding effects due to population structure, and is especially well suited for the mapping of multivariate single cell interaction eQTL. 
The model can handle hundreds of cell states and conditions, and it can be applied to large cohorts of hundreds of thousands of cells for hundreds of  individuals.


\section{Model} 

\begin{equation}\label{eq:scStructLMM}
 \mathbf{y} =  \mathbf{W}\boldsymbol{\alpha} + \mathbf{g}\beta_G + \mathbf{g} \otimes \boldsymbol{\beta_{GxE}} + \mathbf{e} + \mathbf{u} + \boldsymbol{\epsilon} 
\end{equation}

Because we use Rao's score test, we only need to evaluate the likelihood under $H_0$:

\begin{equation}
 \mathbf{y}|H_0 =  \mathbf{W}\boldsymbol{\alpha} + \mathbf{g}\beta_G + \mathbf{e} + \mathbf{u} + \boldsymbol{\epsilon} 
\end{equation}

Where $\mathbf{e} \sim N(\mathbf{0},\sigma_e^2 \mathbf{E}\mathbf{E}^T)$, $\mathbf{u} \sim N(\mathbf{0},\sigma_g^2 \mathbf{G}\mathbf{G}^T)$ and $\boldsymbol{\psi} \sim N(\mathbf{0},\sigma_n^2 \mathbf{I_N})$.\\

Now to be able to use the trick described in Section 2.3.3, we need to write the Covariance matrix in the form $\sigma^2(\mathbf{M}+\delta\mathbf{I})$.
To do so, we introduce a weight parameter $\rho_1$ such that:

\begin{equation}
\begin{split}
    Cov(\mathbf{y}) = \sigma_e^2 \mathbf{E}\mathbf{E}^T + \sigma_g^2 \mathbf{G}\mathbf{G}^T+ \sigma_n^2 \mathbf{I} =\\
    \sigma_k^2[\rho_1\mathbf{E}\mathbf{E}^T + (1-\rho_1) \mathbf{G}\mathbf{G}^T] + \sigma_n^2 \mathbf{I} =\\ \sigma_k^2\{\boldsymbol{\Sigma}(\rho_1) + \delta_1 \mathbf{I}\}
\end{split}
\end{equation}

Where $\sigma_k^2\rho_1 = \sigma_e^2$ and $\sigma_k^2(1-\rho_1) = \sigma_g^2$;
$\delta_1 = \sigma_n^2/\sigma_k^2$ and $\boldsymbol{\Sigma}(\rho_1) = \rho_1\mathbf{E}\mathbf{E}^T + (1-\rho_1) \mathbf{G}\mathbf{G}^T$.\\

Note that $\boldsymbol{\Sigma}$ only depends on $\rho_1$. 
To efficiently implement this, we perform a grid search over $\rho_1$ with as little as 10 possible values, thus only slowing down the original method by a factor of 10.

\section{Fast implementation}

To test for GxE interactions we apply the same strategy that is used in the StructLMM method, which in turn adopts the strategy described in Rao et al (ref).
In this section I describe the key steps.

First, we define score statistic Q as:

\begin{equation}
    Q = \frac{1}{2}y^TP (\partial K)P y 
\end{equation}

Where:

\begin{equation}
    P = K^{-1}-K^{-1}W(W^TK^{-1}W)^{-1}W^TK^{-1}
\end{equation}

Now, it can be shown that Q can be expressed as:

\begin{equation}
    Q \sim \sum_i \lambda_i \chi^2_i 
\end{equation}

Where $\lambda_i$'s are the non-zero eigenvalues of $\frac{1}{2}\sqrt{P} \partial K \sqrt{P}$.

Now since the eigenvalues of $AA^T$ are the same as those of $A^TA$, we can re-arrange and compute $\lambda_i$'s as the eigenvalues of:

\begin{equation}
    \frac{1}{2}\sqrt{\partial K} P \sqrt{\partial K}
\end{equation}

instead.
We use the Davies method (ref) to compute the corresponding p values.


\section{Simulation data}

First, we applied the model on simulated data.

\subsection{Simulation strategy}
Briefly, from the full model (eq. 6.1) we simulate only an intercept as covariate:  

\begin{equation}
 \mathbf{y} = \mathbf{y}_0 + \mathbf{g}\beta_G + \mathbf{g} \otimes \boldsymbol{\beta_{GxE}} + \mathbf{e} + \mathbf{u} + \boldsymbol{\epsilon} 
\end{equation}

We column normalize all terms so that the total variance sums to 1.
We set the variance explained by both genetic terms $var_G+var_{GxE}=\sigma_0^2$, and call the rest $v = 1-\sigma_0^2$.
Further, we regulate the amount of variance driven by GxE using an additional weighting factor $\rho_0$, such that: $var_G = (1-\rho_0)\sigma_0^2$ and $var_{GxE} = \rho_0\sigma_0^2$
For simplicity, we set the variances explained by the last three terms to be the same:
$\sigma_E^2 = \sigma_g^2 = \sigma_n^2 = v/3$

\subsection{Calibration analysis}

Using simulated data, we checked that our model was calibrated both in the case of no genetic effects at all (i.e. $\sigma_0^2 = 0$) and in the case of persistent effects only (no GxE, i.e. $\rho_0 = 0$).

\subsection{Comparison with Struct LMM.0}

Next, we compared our model to the original StructLMM model.
StructLMM is expected to have issues in the presence of extended repeated structure, which it is not equipped to deal with.
To address this, we simulated various numbers of repeats per donor - mimicking cells -  ranging from 10 to 500.
Indeed, we observe over-inflation of StructLMM in the presence of many repeats making the model not calibrated (Fig. Xa).
sc-StructLMM, on the other hand, is nicely calibrated (Fig. Xb).

\subsection{Comparison with standard interaction test}

We then performed power analysis when comparing our model with one where the environments are modelled as fixed effects (see Section 2.4.2).

\newpage

\section{Real data}

Next, we applied the model on the data described in Chapter 3.1

\section{Upstream analysis}

One of the key steps in running this method is choosing the environmental factors.
We expect most applications to be for scRNA-seq datasets only (and genotypes).
This means that the environments need to be estimated from the transcriptomic data, which is also used as phenotype, causing possible circularity issues.


A user might have 
PCA
MOFA (single omic)

\section{Downstream analysis} 

\subsection{Estimate cell-specific genetic effects}
One important 

\begin{equation}
    \mathbf{f}(\mathbf{X}) \sim \mathrm{GP}(\mathbf{m}(\mathbf{x}), k(\mathbf{x},\mathbf{x}^T))
\end{equation}

Out of sample prediction for $\mathbf{f}_*$'s best estimator is its BLUP, defined as its expected value condition on $\mathbf{f}$ and $\mathbf{X},\mathbf{X}_*$:

\begin{equation}
    E[\mathbf{f}_*|\mathbf{f}] = \mathbf{m}_* +k(\mathbf{X}_*,\mathbf{X})k(\mathbf{X},\mathbf{X})^{-1}(\mathbf{f}-\mathbf{m})
\end{equation}

In our case,

\begin{equation}
    \mathbf{f}(\mathbf{X}) = \mathbf{y} = \mathbf{W}\boldsymbol{\alpha}+\mathbf{g}\beta_G+\mathbf{g}\boldsymbol{\beta}_{GxE}+\mathbf{e} + \mathbf{u} + \boldsymbol{\psi}
\end{equation}

with ($\mathbf{X} = {\mathbf{W},\mathbf{g},\mathbf{E},\mathbf{K}}$):

\begin{equation}
    \mathbf{m}(\mathbf{X}) = \mathbf{W}_*\boldsymbol{\alpha}_{*}+\mathbf{g}_*\beta_G
\end{equation}

\begin{equation}
    k(\mathbf{X},\mathbf{X}) = \sigma_{GxE}^2(\mathbf{g}\otimes\mathbf{E})(\mathbf{g}\otimes\mathbf{E})^T
\end{equation}

\begin{equation}
\mathbf{y}_{*}^{BLUP} = E[\mathbf{y}_*|\mathbf{y}] = 
\mathbf{W}_*\boldsymbol{\alpha}_{*}+\mathbf{g}_*\beta_G+\mathbf{E}_*\gamma + 
k(\mathbf{X}_*\mathbf{X})K^{-1}(\mathbf{y}-\mathbf{W}\boldsymbol{\alpha}-\mathbf{g}\beta_G-\mathbf{E}\gamma)
\end{equation}

\section{Discussion}

\begin{itemize}
    \item GLMM (Poisson, NB)
    \item additional covariances?
    \item other applications (longitudinal data)
\end{itemize}
