%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Fifth Chapter *********************************
%*******************************************************************************

\chapter{Population-scale differentiation of iPSCs to a neuronal fate}

The study described in Chapter 4 acted as a proof of principle study where we ..

In this second study, we scale up .. and apply similar principles to a much longer and more complex differentiation protocol, considering iPSCs differentiating towards a midbrain neuronal fate.\\

First, the use of the droplet-based scRNA-seq technology allows us to assay a much larger number of cells providing an overview of the plethora of brain cell types generated by this protocol. 
Moreover, the larger number of cell lines included and the longer protocol allows us to dive deeper into the differences across lines in efficiency to differentiate, and allows us to start exploring possible causes.
Finally, the closer resemblance of the differentiated cells to primary tissues allows to start looking into the effects of disease and trait associated variants onto specific cell types and across development. \\

Briefly, the dataset we describe in this chapter is genome-wide single cell RNA-sequencing profiling of over one Million differentiating iPS cells collected from cell lines from 215 healthy donors. 
Data is collected at three maturation stage following differentiation to midbrain dopaminergic neurons: progenitor-like state (day11), young neurons (day30), and more mature neurons (day52). 
Additionally, just before the latest time point half of the cells were stimulated with rotenone, to simulate oxidative stress. 

\newpage

\begin{Abstract}
% \subsection{Contributions}

\hspace{-3mm}\textbf{Contributions} This work is the result of a very productive collaboration between the Stegle, Merkle, Marioni and Gaffney labs, which was funded by Open Targets \cite{}.
The data was generated by Dan Gaffney’s lab at the Wellcome Trust Sanger Institute, and the experiments were largely led by Julie Jerber, who also contributed to the interpretation of the results. 
The statistical methods and analyses described in this chapter were co-supervised by Dan Gaffney and Oliver Stegle. 
Daniel Seaton processed the data and performed quality control (QC). 
Daniel and I also developed and implemented the statistical methods under the supervision of Oliver Stegle and Dan Gaffney with some input from Florian Merkle, John Marioni and Natsuhiko Kumasaka.
In particular, Natsuhiko performed the colocalisation analysis.
The code for processing, analysing and plotting the data is open source and freely accessible here: https://github.com/single-cell-genetics/singlecell\_neuroseq\_paper.
Julie Jerber, Daniel Seaton, Florian Merkle, Dan Gaffney and Oliver Stegle and I wrote the manuscript, with input from Natsuhiko Kumasaka and John Marioni.
A preprint \cite{jerber2020population} can be found on biorxiv: https://www.biorxiv.org/content/10.1101/2020.05.21.103820v1, as:\\

Julie Jerber*, Daniel D. Seaton*, Anna S.E. Cuomo*, Natsuhiko Kumasaka, James Haldane, Juliette Steer, M Patel, D Pearce, M Andersson, Marc Jan Bonder, Ed Mountjoy, Maya Ghoussaini, Madeline A. Lancaster, the HipSci Consortium, John C. Marioni, Florian T. Merkle, Oliver Stegle, Daniel J. Gaffney. Population-scale single-cell RNA-seq profiling across dopaminergic neuron differentiation, 2020 (* equal contributions).

\end{Abstract}

\section{Introduction}

As discussed, genetic variation can significantly alter cell function, for example by altering gene expression. 
Human iPSCs are a promising cellular model for assessing the cellular consequences of human genetic variation across different lineages, developmental states and cell types. 
In particular, human iPSCs facilitate the study of developmental time points and stimulation conditions that would be challenging to obtain \textit{in vivo}. 
The creation of cell banks containing hundreds of iPSC lines 1 provides an exciting opportunity to carry out pop ulation-scale studies in vitro 2-5 \cite{cuomo2020single, strober2019dynamic, schwartzentruber2018molecular, alasoo2018shared}.
However, differentiating iPSCs is expensive and labour-intensive, and differentiation experiments are difficult to compare due to substantial batch variation. 
Thus, studies of more than a handful of lines remain a significant challenge.
Furthermore, most iPSC differentiation protocols produce a heterogenous population of cells of which the target cell type is a subset 6–8 \cite{d2019vitro, banovich2018impact, volpato2018reproducibility, nguyen2018single}. 
This variability in differentiation outcomes hinders efforts to dissect the genetic contributions to cellular phenotypes.\\

Single cell sequencing has enabled “multiplexed” experimental designs, where cells from multiple donors are pooled together 2,9 \cite{cuomo2020single, nguyen2018single}. 
Pooling improves throughput and allows experimental variability between differentiation batches to be rigorously controlled, by enabling cell type heterogeneity to be accounted for in downstream analysis. 
To date, multiplexed experimental designs have only been applied to short differentiation protocols (over a period of days), that generate cells corresponding to very early stages of development, and have not captured developmental progression toward a mature cell fate. 
Population-scale pooling during long-term differentiation offers the opportunity to examine the effect of common genetic variants on gene expression in each cell population produced over neural development, providing a foundation for future mechanistic studies.

Here, we develop and apply a multiplexing strategy to profile the differentiation and maturation of more than two hundred iPSC lines derived from the Human Induced Pluripotent Stem Cell Initiative (HipSci) towards a midbrain neural fate, including dopaminergic neurons (DA). 
DA are involved in motor function and other cognitive processes and play key roles in neurological disorders, including Parkinson’s Disease (PD) 10,11 \cite{osborn2017seq, stoddard2020stem}. 
To study how these cells differentiate, and how genetic background could influence differentiation, we employed a well-established protocol 12 and collected cells at three maturation stages (progenitor-like, young neurons, and more mature neurons), covering 52 days of differentiation. 
We additionally exposed cells on day 51 to rotenone, to explore how genetic variation shapes the neuronal response to oxidative stress. 
Using this system, we create the first map of expression quantitative trait loci (eQTL) at multiple stages of human neuronal differentiation, and identify nearly 500 novel trait / eQTL colocalisations. 
Using estimates of cell population composition based on single cell RNA-seq, we demonstrate that a strong, cell intrinsic-differentiation bias affects a significant proportion of iPSC lines, such that approximately 25\% reproducibly fail to produce any neuronal cells.\\

longer iPSC differentiation protocol

relevant for cell therapy (dopaminergic neurons and PD)

different 

\section{Results}

\subsection{Data overview}

\subsection{Line-to-line variation in differentiation efficiency}

\subsection{iPSC gene expression signatures predict neuronal differentiation efficiency}

\subsection{eQTL mapping in neuronal cell types}

Finally, we mapped eQTL in the cell types from the dataset described in the second part of Chapter 3 (3.2).
Here our focus was on understanding how individual-to-individual genetic variation influenced gene expression across these cell types during differentiation and in response to stimulation.
Specifically, we mapped cis expression quantitative trait loci (eQTL) separately for each of the 14 distinct cell populations that corresponds to the profiled “cell type”-“condition” contexts (fig.). 
eQTL were mapped by calculating aggregate expression levels for each donor, considering common gene-proximal variants (MAF>0.05, plus or minus 250 kb around genes; Methods). 
Variability in differentiation efficiency between lines resulted in substantial differences in the number of cells collected for each donor (Supplementary Fig. 8a), affecting accuracy of the estimates of aggregated expression. 
To account for this source of noise, we adapted commonly used eQTL mapping strategies2 based on linear mixed models (LMMs) by incorporating an additional variance component into the model (Methods). 
This approach greatly increased the power to map eQTL, resulting in a total of 4,087 genes with at least one eQTL in any of the contexts (hereafter “eGene”, FDR < 5\%, Fig. 4a, Supplementary Fig. 8b, Supplementary Table 7).


In Chapter 4 we have focused on reproducing standard "mean-level" expression level eQTL mapping using scRNA-seq, where the phenotype of interest is expression abundance within a homogeneous population of cells.
We can call such efforts "pseudo-bulk" approaches, where we are essentially replicating bulk-like expression values and performing the eQTL test adapting approaches used for traditional eQTL mapping using bulk RNA-seq. 
In the applications we and others have described \cite{van2018single,cuomo2020single}, the value of using scRNA-seq lies in the fact that we are able to, within a single experiment, unbiasedly define and assess multiple different cell types, whilst retaining a single cell resolution.\\

In this chapter we want to show examples of eQTL mapping using different phenotypes, where we specifically exploit the single cell resolution.

Interaction eQTL can be though of as a sort of GxE effect, where the effect of a genetic variant on the expression of a given gene is modulated by another factor.
If in traditional (GWAS) GxE analysis the factor is often an environmental variable such as diet or exercise, in eQTL we are more often looking at molecular changes, such as cell type composition, or the level of activation due to some stimulus, etc..

Using bulk:

Sometimes, other genes can be used directly as factors, like in \cite{zhernakova2017identification}

However, if we want to identify molecular changes it only makes sense to assess expression at the single cell level, and truly identify continuous states..

% \subsection{other results}

As discussed in (2.4) there are multiple ways to test for interaction effects, where variation in the phenotype is affected by a combined effect of the genotype at a certain locus and some environmental component.\\

In the context of eQTL mapping those environments are more often called cell states or types and are typically estimated from the expression profiles themselves. 
Then an interaction eQTL might be an eQTL whose strength changes continuously over some differentiation trajectory for example, or an eQTL whose strength varies from cell type to cell type or from one cell cycle phase to another.\\ 

In some sense, we have already performed a related analysis: by subdividing cells into developmental stages and cell types prior to eQTL mapping, we have essentially performed stratified tests (see section 2.4.1).
In the neuronal differentiation study, we first assigned cells to cell types using marker genes, and then mapped eQTL in each cell type separately.
In the endoderm study, we ordered our cells along a computationally inferred pseudotime, and then essentially binned cells in three groups, again reflecting known biology of this developmental process and using marker genes.\\

However, the single cell resolution allows to assess more subtle changes, by performing a joint analysis on all cells and measuring the effect of differentiation taken as a continuum.
In the next section, we show different approaches we used to perform interaction eQTL mapping in the endoderm differentiation data.

\subsection{Colocalization of eQTL with disease risk variants}

\section{Discussion}